### ISSUE TITLE
Webhooks Not Triggering

### ISSUE THREAD

**Issue by larissalamaster** [merchantprotocol/theadslab#55]
I set up a few webhooks yesterday and they haven't triggered at all. The sites have visitors coming to them so it should be able to identify contacts, but nothing has triggered on zapier.
--
When was this bug experienced: today/yesterday (5/17-5/18)
What is the purpose of this issue? The purpose is that we can't get any data out of Ads Lab now (Dashboard visitor id isn't working, Google Sheets integration isn't working, now Webhook isn't working). 

--
Attach any screenshots, screencasts, customer information, chat transcripts, emails to support the debugging effort.

![Screen Shot 2022-05-18 at 11 04 59 AM](https://user-images.githubusercontent.com/105022789/169113058-cc4346a6-8997-44c2-a8d6-3d5bcbd16dae.png)

![Screen Shot 2022-05-18 at 11 08 02 AM](https://user-images.githubusercontent.com/105022789/169114010-5dc1e436-7b1f-4fe4-b48c-0b0a3cf5dd04.png)

**Comment by abadger13** [comment]
@larissalamaster in anticipation of this working (I'm trying to set up web hooks as well). Did you find a spreadsheet that has the header data in your list? Where did you find that for the proper formatting? Thanks in advance.

**Comment by jonathonbyrdziak** [comment]
@larissalamaster checkin in on this. is it still not firing for you?

**Comment by abadger13** [comment]
I get no data either. Empty spreadsheet. All tabs.

**Comment by jonathonbyrdziak** [comment]
> I get no data either. Empty spreadsheet. All tabs.

that sounds like a google sheet sync issue, not a webhook issue

**Comment by abadger13** [comment]
> > I get no data either. Empty spreadsheet. All tabs.
> 
> that sounds like a google sheet sync issue, not a webhook issue

I agree. But since the webhook appears to be working (I'm in integromat) and the google sheet doesn't show data whether in the webhook integration or in the google sheet integration, I'm at a loss. Still can't pull out data for clients.
